src/main/scala/io/github/pauljamescleary/petstore/repository/DoobiePetRepositoryInterpreter.scala

```diff
@@ -76,12 +76,26 @@ class DoobiePetRepositoryInterpreter[F[_]: Monad](val xa: Transactor[F])
       .list
       .transact(xa)
 
-  override def findByStatus(statuses: NonEmptyList[PetStatus]): F[List[Pet]] = {
-    val q = sql"""SELECT NAME, CATEGORY, BIO, STATUS, TAGS, PHOTO_URLS, ID
+  def findByStatus(statuses: NonEmptyList[PetStatus]): F[List[Pet]] =
+    (sql"""SELECT NAME, CATEGORY, BIO, STATUS, TAGS, PHOTO_URLS, ID
```

Comment:
do we need the parens?

Reply:
The parens are needed to join the SQL interpolator with the fragment.
```
    (sql"""SELECT NAME, CATEGORY, BIO, STATUS, TAGS, PHOTO_URLS, ID
            FROM PET
           WHERE """ ++ Fragments.in(fr"STATUS", statuses))
      .query[Pet]
      .list
      .transact(xa)
```